liquibase-service/src/main/resources/db/rdb/tables/017-create_nrt_observation-002.sql

```diff
@@ -0,0 +1,30 @@
+IF NOT EXISTS(SELECT 1 FROM sys.columns   WHERE Name = N'ctrl_cd_display_form'   AND Object_ID = Object_ID(N'nrt_observation'))
+    BEGIN
+        ALTER TABLE nrt_observation
+            ADD ctrl_cd_display_form varchar(20);
+    END;
+
+IF NOT EXISTS(SELECT 1 FROM sys.columns   WHERE Name = N'obs_domain_cd_st_1'   AND Object_ID = Object_ID(N'nrt_observation'))
```

Comment:
For Alter statements, it should be `IF EXISTS`

Reply:
It should be like this: 
Alter the table only if the above table exists

Reply:
Ah I see. I am running an IF NOT EXISTS on the column level, but on the table level, it will be IF EXISTS.